stan/math/rev/mat/functor/cvodes_integrator.hpp

```diff
@@ -134,12 +112,15 @@ class cvodes_integrator {
     if (cvodes_mem == NULL)
       throw std::runtime_error("CVodeCreate failed to allocate memory");
 
+    const size_t size
```

Comment:
it looks like this is just used in line 119. Mind either:
- renaming to `coupled_size` and removing the comment
or
- just use it directly in line 119?

Reply:
Sure, will do.